### DISCUSSION TITLE
$ilike, $like not working

### DISCUSSION THREAD

**Discussion by toplinuxsir** [prest/prest#689] [API Server] [answered]
**Describe the bug**

$ilike and $like not working,
The query url:

I have a table: t_product_type
and query for it :
http://127.0.0.1:8081/smart_erp3/public/t_product_type?prd_barcode='$ilike.B%25'
return nothing
The debug generated sql is 
```
SQL:SELECT json_agg(s) FROM (SELECT * FROM "smart_erp3"."public"."t_product_type" WHERE "prd_barcode" ILIKE $1 ) s parameters: ['BX%']
```

But i  run the sql above in db console ,
It returns data.

**Comment by avelino** [comment, marked as answer]
you are using `'` in the wrong place, your URI should be

`/smart_erp3/public/t_product_type?prd_barcode=$ilike.B%25`

this query should return all records that `prd_barcode` starts with `B` or `b`

**Comment by toplinuxsir** [comment]
@avelino Thanks it woks!
